src/xdocs/property_types.xml

```diff
@@ -831,16 +831,62 @@
 
     <section name="SeverityLevel">
       <p>
-        This type represents the severity level of a check violation. The
-        valid options are:
+        This type represents the severity level of a module violation.
+        The following table describes the valid options:
       </p>
 
-      <ul>
-        <li><code>ignore</code></li>
-        <li><code>info</code></li>
-        <li><code>warning</code></li>
-        <li><code>error</code></li>
-      </ul>
+      <div class="wrapper">
+        <table>
+          <tr>
+            <td>Option</td>
+            <td>Definition</td>
+          </tr>
+
+          <tr>
+            <td><code>ignore</code></td>
+            <td>
+              If a violation occurs, modules with this severity are ignored as if they never
+              occurred.
+              Checkstyle will not display the violation and will not fail the build.
```

Comment:
I have some worries that we need to phrase it as "Checksyle's Audit Listeners (reports)" as decision on what to show and how if up to module that does report generation.